maestro-driver-ios/maestro-driver-iosUITests/maestro_driver_iosUITests.swift

```diff
@@ -18,44 +18,12 @@ class maestro_driver_iosUITests: XCTestCase {
 
     func testHttpServer() async throws {
         let server = HTTPServer(address: .loopback(port: 22087))
-        let subTreeRoute = HTTPRoute(Route.subTree.rawValue)
-        let runningAppRoute = HTTPRoute(method: .POST,
-                                        path: Route.runningApp.rawValue)
-        let swipeRoute = HTTPRoute(method: .POST,
-                                   path: Route.swipe.rawValue)
-        let inputTextRoute = HTTPRoute(method: .POST, path: Route.inputText.rawValue)
-        let touchRoute = HTTPRoute(method: .POST, path: Route.touch.rawValue)
-        let screenshotRoute = HTTPRoute(Route.screenshot.rawValue)
-        let isScreenStaticRoute = HTTPRoute(Route.isScreenStatic.rawValue)
-        await server.appendRoute(subTreeRoute) { request in
-            let handler = RouteHandlerFactory.createRouteHandler(route: .subTree)
-            return try await handler.handle(request: request)
-        }
-        await server.appendRoute(runningAppRoute) { request in
-            let handler = RouteHandlerFactory.createRouteHandler(route: .runningApp)
-            return try await handler.handle(request: request)
-        }
-        await server.appendRoute(swipeRoute) { request in
-            let handler = RouteHandlerFactory.createRouteHandler(route: .swipe)
-            return try await handler.handle(request: request)
-        }
-        await server.appendRoute(inputTextRoute) { request in
-            let handler = RouteHandlerFactory.createRouteHandler(route: .inputText)
-            return try await handler.handle(request: request)
-        }
-        await server.appendRoute(touchRoute) { request in
-            let handler = RouteHandlerFactory.createRouteHandler(route: .touch)
-            return try await handler.handle(request: request)
-        }
-        await server.appendRoute(screenshotRoute) { request in
-            let handler = RouteHandlerFactory.createRouteHandler(route: .screenshot)
-            return try await handler.handle(request: request)
-        }
-        await server.appendRoute(isScreenStaticRoute) { request in
-            let handler = RouteHandlerFactory.createRouteHandler(route: .isScreenStatic)
-            return try await handler.handle(request: request)
-
+        
+        for route in Route.allCases {
+            let handler = RouteHandlerFactory.createRouteHandler(route: route)
+            await server.appendRoute(route.toHTTPRoute(), to: handler)
         }
+        
```

Comment:
Can this code move to a new struct `XCTestHTTPServer` or similar?
The `Routes` enum can then live in / next to that struct.